src/datadog_api_client/v1/api/hosts_api.py

```diff
@@ -165,9 +175,24 @@ def get_host_totals(self, **kwargs):
         :type _from: int, optional
         :rtype: HostTotals
         """
+        kwargs: Dict[str, Any] = {}
+        if _from is not unset:
+            kwargs["_from"] = _from
+
         return self._get_host_totals_endpoint.call_with_http_info(**kwargs)
 
-    def list_hosts(self, **kwargs):
+    def list_hosts(
+        self,
+        *,
```

Comment:
What is this `*,`  ?

Reply:
It means that parameters after it are keywords only args. See https://peps.python.org/pep-3102/.